.gitignore

```diff
@@ -7,4 +7,7 @@ dist/*
 .venv
 .env/
 Pipfile
-*.egg-info
\ No newline at end of file
+*.egg-info
```

Comment:
David this code is awesome and elegant-- great use of a variable.
Choosing sensible names will save you time and energy later. You’ll be able to figure out, from the name, what a certain variable, function, or class represents. You’ll also avoid using inappropriate names th...

📄 [Realpython](https://realpython.com/python-pep8/#naming-conventions)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Elegant